docs/setup.md

```
@@ -114,12 +114,8 @@ Cloudflare OneにIdentity Providerの登録(`/integrations/identity-providers`)
 
 ## バージョン更新
 
-```sh
-$ sh update.sh
-```
-
 ```sh
 $ git pull origin main
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)